view/frontend/templates/js/replacejs.phtml

```diff
@@ -35,6 +35,62 @@ ob_start();
     // in payment method renderer, vendor/boltpay/bolt-magento2/view/frontend/web/js/view/payment/method-renderer/boltpay.js
     window.boltConfig = <?php echo $block->getSettings(); ?>;
 
+    var trackCallbacks = {
```

Comment:
question: while you are at it wdyt about creating a php function to output the following js so we dont have that code all repeated. The php fn takes `$trackCallbacks['checkout_start']` as input

```
function() {
  try {
    ...callback_here...
  } catch (error) {
    console.error(error);
  }
}
```

if you have a different plan for future PRs ignore this

Reply:
done